prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt

```diff
@@ -429,4 +447,36 @@ object CustomFHIRFunctions {
             mutableListOf(StringType(result))
         }
     }
+
+    /**
+     * Applies a timezone given by [parameters] to a date in [focus] and returns the result
+     * @return a date in the new timezone
+     */
+    fun changeTimezone(
+        focus: MutableList<Base>,
+        parameters: MutableList<MutableList<Base>>?
+    ): MutableList<Base> {
+        if (focus.size != 1) {
+            throw SchemaException("Must call changeTimezone on a single element")
+        }
+
+        val inputDate = FhirBundleUtils.convertFhirType(focus[0], focus[0].fhirType(), "dateTime") as? DateTimeType
+            ?: throw SchemaException("Must call changeTimezone on a dateTime compatible value")
```

Comment:
This message may be confusing. What is a dateTime compatible value? Is that defined in FHIR or FHIR Path? Simplify